package.json

```diff
@@ -69,6 +69,7 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "react-select": "^3.1.0",
+    "react-hook-form": "^6.11.0",
```

Comment:
I don't think this should be a long-term dependency. This requires us to manage two installations of `react-hook-form`.